src/pyseekdb/client/client_base.py

```
@@ -804,26 +873,115 @@ def get_collection(
+        """
+        Validate embedding function
+
+        Args:
+            embedding_function: Embedding function user provided
+            embedding_function_persistence: Embedding function restored from table metadata
+
+        Returns:
+            Embedding function
+        """
+
+        if embedding_function_persistence is not None and not embedding_function is _NOT_PROVIDED:
```

Ruff (0.14.13)

Test for object identity should be `is not`

Convert to `is not`

(E714)